custom/icds_reports/reports/service_delivery_dashboard.py

```diff
@@ -81,6 +81,12 @@ def base_data(row_data):
             )
 
     for row in data:
+        if age_sdd == '0_3':
+            if all([row['valid_visits'] != DATA_NOT_ENTERED,
+                    row['expected_visits'] != DATA_NOT_ENTERED,
+                    row['valid_visits'] > row['expected_visits']]):
+                row['valid_visits'] = row['expected_visits']
```

Comment:
I did not get a chance to review this before it was merged but this logic is wrong. It will lead to both undercounting of visits and inconsistent data at various levels. For example, imagine a sector with 2 AWWs, one who did 5/3 and one who did 1/3. At the AWW level, the AWW who did 5 visits will only have gotten credit for 3, but at the supervisor level it will say 6/6 even though the numbers shown at the lower level will not add up.